common/lib/xmodule/xmodule/css/video/display.scss

```diff
@@ -366,18 +316,15 @@
       .control {
         @extend %video-control;
 
-        .icon-fallback-img {
-
-          .icon {
-            // if the icon font doesn't render, we need to provide dimensions for the svg's
-            width: 1em;
-            height: 1em;
+        .icon {
```

Comment:
The icons should inherit the same font size as the container it's in, so it matches the text it goes with. Can you maybe use `100%` instead then, so it's a bit more clear (although I realize `1em` is basically the same).